config/v1/types_tlssecurityprofile.go

```diff
@@ -107,6 +110,8 @@ type TLSSecurityProfile struct {
 	//     - ECDHE-ECDSA-AES128-GCM-SHA256
 	//   minTLSVersion: TLSv1.1
 	//
+	// NOTE: Currently unsupported.
+	//
```

Comment:
This addition is in the godoc for `Custom`, but it should be in the godoc for `Modern`.

Reply:
Thanks, I did this in a hurry and missed the right spot